xesmf/backend.py

```diff
@@ -18,7 +18,10 @@
 import os
 import warnings
 
-import ESMF
+try:
+    import esmpy as ESMF
+except:
```

Comment:
```suggestion
except ImportError:
```
Should work the same and satisfy the linter.

Reply:
thanks! will try it out now